faker/seeder.js

```diff
@@ -0,0 +1,79 @@
+const faker = require('faker');
+const mysql = require('mysql');
+const db = require('../server/db/index.js');
+
+
+let randomItems = [];
+let randomImages = [];
+
+//populates arrays with random objs
+for (let i = 0; i < 100; i++) {
+  let itemObj = {
+    id: i,
+    itemNum: faker.random.number(),
+    POSNum: faker.random.number(),
+    avgScore: faker.random.number(5),
```

Comment:
I'm just starting to do the seeding myself, so I'm not 100% if this is right or not, but I think you can set up a range of numbers. Relating to that, do you want a minimum value of 1 on both this and reviewCount below? Or do you want the random number range to also include 0?